From 79457fe6efdc4f3d5ad6b7d365790e12bf7b6d7d Mon Sep 17 00:00:00 2001
From: Sylvain Pineau <sylvain.pineau@canonical.com>
Date: Mon, 12 Sep 2016 14:58:40 +0200
Subject: [PATCH] patch to load credentials

---
 process-merge-requests | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/process-merge-requests b/process-merge-requests
index 633086f..ce9212d 100755
--- a/process-merge-requests
+++ b/process-merge-requests
@@ -289,7 +289,9 @@ def main():
         help=_("name of the launchpad project to process"))
     ns = parser.parse_args()
     _logger.info(_("Logging into launchpad.net"))
-    lp = Launchpad.login_with('process-merge-requests', ns.lp_api_url)
+    lp = Launchpad.login_with(
+        'process-merge-requests', ns.lp_api_url,
+        credentials_file='/home/ubuntu/.config/pmr/credentials')
     _logger.info(_("Checking for new things to land every minute..."))
     try:
         while True:
-- 
2.7.4

